src/Avalonia.Controls/Automation/Peers/NoneAutomationPeer.cs

```diff
@@ -0,0 +1,28 @@
+using Avalonia.Automation.Platform;
+using Avalonia.Controls;
+
+#nullable enable
+
+namespace Avalonia.Automation.Peers
+{
+    /// <summary>
+    /// An automation peer which represents an element that is exposed to automation as non-
+    /// interactive or as not contributing to the logical structure of the application.
+    /// </summary>
+    public class NoneAutomationPeer : ControlAutomationPeer
+    {
+        public NoneAutomationPeer(IAutomationNodeFactory factory, Control owner)
+            : base(factory, owner) 
+        { 
+        }
+
+        protected override AutomationControlType GetAutomationControlTypeCore()
+        {
+            return AutomationControlType.Group;
```

Comment:
This does not sound right, "None" content is a group? How about just removing it from UIA tree then instead of saying it is a group?

Reply:
The problem with completely removing them from the OS-level automation tree is that it breaks the `AutomationPeer` -> `AutomationNode`  1:1 mapping and means that we have to have _another_ level of abstraction to handle parent/child relationships in the case where peers don't map directly to OS automation nodes. We've already got 3 trees to keep in sync so at least if the nodes in the "peer" tree match 1:1 with the nodes in the OS tree it reduces some complexity.

Win32 and OSX provide a facility for "hiding" controls at the automation node level (`UIA_IsControlElementPropertyId`/[`accessibilityElement`](https://developer.apple.com/documentation/appkit/nsaccessibility/1535002-accessibilityelement?language=objc)), which looked to me to be sufficient?

Maybe instead of using `Group` we should have a new `AutomationControlType.None` type which will be mapped to the most relevant type for "ignored" elements at the OS level?

Reply:
Ah I see. In that case, group is fine, having a "none" control type would be really cool, however I'm not sure how much of a benefit there would be.